src/displayapp/WatchFaces.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+namespace Pinetime {
+  namespace Applications {
+    enum class WatchFace : uint32_t {
```

Comment:
By using uint8_t we should be able to avoid updating the settings version and avoid an unnecessary settings reset.

Reply:
Yep, and to do that we would also need to explicitly give the enumerations values like with WakeUpMode, because otherwise the values are undefined.

Reply:
`CheckboxList` uses `uint32_t`. I still changed the enum to `uint8_t` since it should be fine.